eyeball-im-util/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-# 0.8.0
+# 0.9.0
 
 - Upgrade `eyeball-im` dependency to 0.7
+- Add the `Skip` adapter
```

Comment:
This should be part of the 0.8.1 changelog, not 0.9.0.

Reply:
Right!